src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -5,40 +5,72 @@
 import com.fasterxml.jackson.databind.deser.DeserializationProblemHandler;
 
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.RobotMap;
+import frc.robot.commands.SingleShotCommand;
+import frc.robot.controllers.TalonSettings;
+import frc.robot.controllers.TalonSettingsBuilder;
 import frc.robot.controllers.TalonSpeedController;
 import frc.robot.logger.DataLoggerFactory;
+import frc.robot.newPoses.FieldPose;
+import frc.robot.newPoses.RobotPose;
 
-
-
- public class IntakeSubsystem extends BaseSubsystem{
+public class IntakeSubsystem extends BaseSubsystem {
     private double INTAKE_SPEED = 1;
 
     private final WPI_TalonSRX intakeMotor = new WPI_TalonSRX(RobotMap.CAN.INTAKE_MOTOR);
     private TalonSpeedController intakeMotorController;
 
     @Override
     public void initialize() {
-        TalonSettings motorSettings = TalonSettingsBuilder.defaults()
-            .withCurrentLimits(20, 15, 200)
-            .brakeInNeutral()
-            .withDirections(false, false)
-            .noMotorOutputLimits()
-            .noMotorStartupRamping()
-            .useSpeedControl()
-            .build();
-            
+        TalonSettings motorSettings = TalonSettingsBuilder.defaults().withCurrentLimits(20, 15, 200).brakeInNeutral()
+                .withDirections(false, false).noMotorOutputLimits().noMotorStartupRamping().useSpeedControl().build();
+
         intakeMotorController = new TalonSpeedController(intakeMotor, motorSettings);
         intakeMotorController.configure();
-        intakeMotor.set(ControlMode.PercentOutput,0);
+        intakeMotor.set(ControlMode.PercentOutput, 0);
     }
 
     public void setIntakeMotorSpeed(double desiredSpeed) {
         logger.log("Intake Motor speed", desiredSpeed);
         this.INTAKE_SPEED = desiredSpeed;
         updateIntakeMotor();
     }
+
     public void updateIntakeMotor() {
         intakeMotorController.setDesiredSpeed(this.INTAKE_SPEED);
     }
+
+    public Command start(){
+        return new SingleShotCommand(this){        
+            @Override
+            public void doCommand() {
+                setIntakeMotorSpeed(1);;
```

Comment:
i would rather see some constants ( FULL_SPEED_FWD, FULL_SPEED_BWD) instead of 1 and -1